### PR TITLE
Update according to latest discussions

### PR DESCRIPTION
**Pull request by ilevkivskyi**
In this PR:
- updates according to latest discussions;
- some shortening and formatting;
- added a short version of 'big concepts';
- added two missing points to rejected ideas.

@kirbyfan64 @phouse512 @gvanrossum Please review.

### CODE REVIEW

**Comment by gvanrossum** [comment]
Hopefully someone not me can review and merge this!

File: pep-0526.txt [hunk 1]
```diff
@@ -231,14 +240,20 @@ any valid assignment target::
   class Cls:
       pass
 
-  c = Cls
-  c.x: int = 0  # Annotates ``c.x`` with ``int``..
+  c = Cls()
```
Comment (refi64) [inline pep-0526.txt:243]: Sorry, my bad!

File: pep-0526.txt [hunk 2]
```diff
@@ -231,14 +240,20 @@ any valid assignment target::
...
+  c.x: int = 0  # Annotates c.x with int.
   c.y: int  # Invalid syntax: no initial value was specified!
 
   d = {}
-  d['a']: int = 0  # Annotates ``d['a']`` with ``int``.
+  d['a']: int = 0  # Annotates d['a'] with int.
   d['b']: int  # Invalid again.
 
+Note that even ``(my_var)`` considered an expression, not a simple name.
```
Comment (refi64) [inline pep-0526.txt]: Missing **is**.

File: pep-0526.txt [hunk 3]
```diff
@@ -287,11 +308,9 @@ Changes to standard library and documentation
 Runtime effects of type annotations
 ===================================
 
-As stated under "Variable Annotations", annotating a local variable will cause
+Annotating a local variable will cause
 the interpreter to treat it as a local, even if it was never assigned to.
-
-If a variable annotation is for a local variable, the annotation will not be
-evaluated::
+Annotation for a local variable will not be evaluated::
```
Comment (refi64) [inline pep-0526.txt]: This sounds weird to me...it should be either "Annotations for locals variables will not be evaluated" or "An annotation for a local variable will not be evaluated".

File: pep-0526.txt (hunk 2 above)
Comment (ilevkivskyi) [reply pep-0526.txt]: Done.

File: pep-0526.txt (hunk 3 above)
Comment (ilevkivskyi) [reply pep-0526.txt]: Done with your first option (Annotations for local variables...)

**Comment by ilevkivskyi** [comment]
@kirbyfan64 Thank you for review! I implemented your comments.

**Comment by lisroach** [comment]
I've noticed that the Non-Goals section has a reference to inspect.getannotations that should be changed, otherwise looks great!

**Comment by ilevkivskyi** [comment]
@lisroach 

> I've noticed that the Non-Goals section has a reference to inspect.getannotations

Thank you! Fixed this.

File: pep-0526.txt [hunk 4]
```diff
@@ -247,9 +262,14 @@ It's illegal to attempt to annotate ``global`` and ``nonlocal``::
...
       global x: int  # SyntaxError
 
+  def g():
+      x: int  # Also a SyntaxError
+      global x
+
 The reason is that ``global`` and ``nonlocal`` don't own variables;
 therefore, the type annotations belong in the scope owning the variable.
 
+Only single assignment target and a single right hand side value is allowed.
```
Comment (phouse512) [inline pep-0526.txt]: sorry to nitpick, but I think `Only single assignment targets and single right hand side values are allowed.` reads a little bit better grammar-wise.

**Comment by phouse512** [comment]
This looks good @ilevkivskyi other than the small grammar thing I noticed.

I can merge after that!

File: pep-0526.txt (hunk 4 above)
Comment (ilevkivskyi) [reply pep-0526.txt]: Thanks!
Fixed this.

**Comment by ilevkivskyi** [comment]
@phouse512 Implemented your comment in a new commit (plus a minor change in the next sentence).

**Comment by phouse512** [comment]
@ilevkivskyi looks good. I will go ahead and merge this now.

**Pull request merged**